### PR TITLE
Rework info in meetings sidebar

### PR DESCRIPTION
**Pull request by cherrybeans**
Now consistent with events sidebar.

screenshot from 2018-01-22 21-38-29

Potential bug: I can't see the accept/decline buttons when making my own events, but I can see them when being invited to a meeting by someone else. Can anyone reproduce? Don't know what's the issue.

### CODE REVIEW

File: app/routes/meetings/components/MeetingDetail.js [hunk 1]
```diff
@@ -123,6 +125,29 @@ class MeetingDetails extends Component<Props> {
...
+      {
+        key: 'Din status',
+        value: `${statusesText[statusMe]}`
+      },
+      {
+        key: 'Når',
+        value: <FromToTime from={meeting.startTime} to={meeting.endTime} />
+      },
+      {
+        key: 'Lokasjon',
```
Comment (edvardvb) [inline app/routes/meetings/components/MeetingDetail.js]: "Sted"? "Lokasjon" sounds kind of forced in Norwegian

Comment (cherrybeans) [reply app/routes/meetings/components/MeetingDetail.js]: Agree. Just decided on not changing the word that was there originally. Will fix.

**Comment by edvardvb** [comment]
I've always been able to see them when I make meetings, and it's kinda nice to not have to go through a link in an email to attend the meeting I just created 😛

**Comment by cherrybeans** [comment]
Just checked in production, and everything is fine! Able to see the buttons on my own meetings.

**Comment by cherrybeans** [comment]
I have noted that in the old meetings it says undefined after "Din status", but not in the new ones.

**Pull request merged**